configs/env-example

```diff
@@ -136,9 +136,11 @@ REDIS_PORT=6379
 REDIS_PASSWORD=
 REDIS_STANDALONE_MODE=true
 QUEUE_CONSUMERS_PORT=
-QUEUE_DASHBOARD_PASSWORD=
-QUEUE_DASHBOARD_USER=
 QUEUE_PREFIX={sims-local}
+# The below secret should be set with any valid base 64 string for local development.
+QUEUE_DASHBOARD_TOKEN_SECRET=
+QUEUE_DASHBOARD_TOKEN_EXPIRATION_SECONDS=3600
```

Comment:
Do we need these props in env-example if local ignores token?

Reply:
They are currently used for local development. We can decide on removing it or the developer can just add a big number to the session that would last for the day.
Making it work locally simular to the deployed version has the benefit of allowing it to be tested in the same way locally.
I would be open either way leaning a bit more for keeping the local behavior close to the prod version.